source/developers/file-transfer-protocol.rst

```diff
@@ -0,0 +1,228 @@
+.. _minio-ftp:
+
+=================================
+File Transfer Protocol (FTP/SFTP)
+=================================
+
+.. default-domain:: minio
+
+.. container:: extlinks-video
+
+   - `File Transport Using FTP and SFTP with MinIO <https://www.youtube.com/watch?v=lNZyL8wD-lI>`__
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+.. versionadded:: MinIO RELEASE.2023-04-20T17-56-55Z
+
+Overview
+--------
+
+You can use the File Transfer Protocol (FTP) or Secure File Transfer Protocol (SFTP) to interact with the objects on a MinIO deployment.
+
+Supported Protocols
+~~~~~~~~~~~~~~~~~~~
+
+When enabled, MinIO supports FTP access over the following protocols:
+
+- SSH File Transfer Protocol (``SFTP``)
+
+  SFTP is defined by the Internet Engineering Task Force (IETF) as an extension of SSH 2.0.
+  SFTP allows file transfer over SSH for use with :ref:`Transport Layer Security (TLS) <minio-tls>` and virtual private network (VPN) applications.
+
+  Your FTP client must support SFTP.
+
+- File Transfer Protocol over SSL/TLS (``FTPS``)
+  
+  ``FTPS`` allows for encrypted FTP communication with TLS certificates over the standard FTP communication channel.
+  ``FTPS`` should not be confused with ``SFTP``, as ``FTPS`` does not communicate over a Secure Shell (``SSH``).
+
+  Your FTP client must support FTPS.
+
+- File Transfer Protocol (``FTP``)
+  
+  Unencrypted file transfer.
+
+  MinIO does **not** recommend using unencrypted FTP for file transfer.
+
+Supported Commands
+~~~~~~~~~~~~~~~~~~
+
+When enabled, MinIO supports the following ``ftp`` operations:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+   :width: 60%
+
+   * - FTP Client Commands
+     - Supported for MinIO
+
+   * - ``get``
+     - supported
+
+   * - ``put``
+     - supported
+
+   * - ``ls``
+     - supported
+
+   * - ``mkdir``
+     - supported
+
+   * - ``rmdir``
+     - supported
+
+   * - ``delete``
+     - supported
+
+   * - ``append``
+     - not supported
+
+   * - ``rename``
+     - not supported
```

Comment:
We could possibly state "MinIO supports only the following ``ftp`` operations and just not talk about anything else.

Reply:
Converted from a table to a list. But I think it's valuable to note we don't support those two specific operation types.